project/MicroService.scala

```diff
@@ -26,8 +26,8 @@ trait MicroService {
     import scoverage.ScoverageKeys
     Seq(
       // Semicolon-separated list of regexs matching classes to exclude
-      ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;model.*;config.*;.*(AuthService|BuildInfo|Routes).*",
-      ScoverageKeys.coverageMinimum := 60,
+      ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;.*(config|views.*|models.*);.*(AuthService|BuildInfo|Routes).*",
```

Comment:
Do we want to exclude all of the `models` package? There is stuff in `NSIUserInfo` which should be tested but also it probably isn't so important to include it in the coverage statistics. The alternative is to have:
```scala
// $COVERAGE-OFF
...
// $COVERAGE-ON$
```
in all the things in `models` which we don't want, but this would be quite annoying